<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8C0LO3K" failures="1" tests="2" name="LoginTest" time="16.43" errors="0" timestamp="2024-06-22T12:47:20 IST">
  <testcase classname="com.demowebshop.tests.LoginTest" name="verifyUserIsAbleToLoginWithValidCred" time="2.981">
    <skipped/>
  </testcase> <!-- verifyUserIsAbleToLoginWithValidCred -->
  <testcase classname="com.demowebshop.tests.LoginTest" name="verifyUserIsAbleToLoginWithValidCred" time="2.031">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.demowebshop.tests.LoginTest.verifyUserIsAbleToLoginWithValidCred(LoginTest.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- verifyUserIsAbleToLoginWithValidCred -->
</testsuite> <!-- LoginTest -->
